sys/Makefile

```diff
@@ -62,6 +62,9 @@ endif
 ifneq (,$(filter fib,$(USEMODULE)))
   DIRS += net/network_layer/fib
 endif
+ifneq (,$(filter fido2_ctap,$(USEMODULE)))
+    DIRS += fido2
+endif
```

Comment:
Does this mean that the `fido2` module is no longer needed? I see it is not modelled in Kconfig (only `fido2_ctap`).

Reply:
The `fido2` module was never needed. I just defined it to have a clear directory structure showing that CTAP is a part of FIDO2 (`fido2/ctap`). The addition to `sys/Makefile` was the wrong fix for an error I got. Fixed this by defining the `FIDO2` module in KConfig as well.